### PR TITLE
#1412 disable league promotion

### PR DESCRIPTION
**Pull request by wsbrenk**
1. changes proposed in this pull request:

disables the league promotion handler. Should be reset when ht server is working again.

2. `src/main/resources/release_notes.md` ...

 - [x] has been updated
 - [ ] does not require update

3. [Optional] suggested person to review this PR @___

### CODE REVIEW

File: src/main/java/module/series/promotion/LeaguePromotionHandler.java [hunk 1]
```diff
@@ -48,12 +43,14 @@ static class DownloadDetails {
...
     public boolean isActive(int seriesId) {
-        List<Integer> supportedLeagues = HttpDataSubmitter.instance().fetchSupportedLeagues();
-        int[] activeWeeks = ModuleConfig.instance().getIntArray("PromotionStatus_ActiveWeeks", new int[] { 14, 15 });
-        int week = HOVerwaltung.instance().getModel().getBasics().getSpieltag();
-        return UserParameter.instance().promotionManagerTest ||
-                (Arrays.stream(activeWeeks).boxed().collect(Collectors.toList()).contains(week) &&
-                        supportedLeagues.contains(seriesId));
+        // TODO: fix HT-Server. As long as it is not working this feature will be disabled
+        return false;
+//        List<Integer> supportedLeagues = HttpDataSubmitter.instance().fetchSupportedLeagues();
```
Comment (tychobrailleur) [inline src/main/java/module/series/promotion/LeaguePromotionHandler.java:48]: Ah sad. :-)

Is it because the HO server is not working any more?

Comment (wsbrenk) [reply src/main/java/module/series/promotion/LeaguePromotionHandler.java:48]: @tychobrailleur yes - the server is not working anymore. i remember some error messages concerning exceeded quotas or something like this.

Comment (wsbrenk) [reply src/main/java/module/series/promotion/LeaguePromotionHandler.java:48]: #1412

Comment (tychobrailleur) [reply src/main/java/module/series/promotion/LeaguePromotionHandler.java:48]: > @tychobrailleur yes - the server is not working anymore. i remember some error messages concerning exceeded quotas or something like this.

Yes, the server that was being used was super-restricted in terms of storage, so not surprised.  As far as I remember, the server-side logic was written in Python by @akasolace, I cannot remember if this was stored in a git repo somewhere.

**Pull request merged**